omod/src/main/webapp/locationMapping.jsp

```diff
@@ -1,5 +1,7 @@
 <%@ include file="/WEB-INF/template/include.jsp" %>
 <%@ include file="/WEB-INF/template/header.jsp" %>
+<openmrs:require anyPrivilege="View Location Mappings,Manage Location Mappings" otherwise="/login.htm"
+                 redirect="/module/dhisconnector/locationMapping.form"/>
```

Comment:
What if the user is already logged in, but doesn't have the above permissions? Then redirecting them into the login page will redirect back to the home page. I think it's better if we could make it more informative.

Reply:
> What if the user is already logged in, but doesn't have the above permissions? Then redirecting them into the login page will redirect back to the home page

yeah @jayasanka-sack. I used the '/login.htm' page because it's the only page that anyone can access. However, the links for these pages are not visible in the navbar. So this redirection will only happen if someone tries to access the page by finding the URL somewhere.

Reply:
Oh, then that's fine.